### ISSUE TITLE
IO Error while requesting logs: org.apache.http.ConnectionClosedException

### ISSUE THREAD

**Issue by theangrydev** [fabric8io/docker-maven-plugin#1369]
### Description
Intermittently, I see the following IO error which makes the plugin fail to start my container:

```
[ERROR]  [DOCKER> IO Error while requesting logs: org.apache.http.ConnectionClosedException: Premature end of chunk coded message body: closing chunk expected Thread-135
[ERROR]  [DOCKER> [ibmcom/mq:9.1.3.0] "ibm-mq-9": Timeout after 20076 ms while waiting on log out 'wait for message'
[ERROR]  [DOCKER> Error occurred during container startup, shutting down...
```

I think the plugin should be more resilient to this kind of IO error and continue polling the logs.

### Info
* d-m-p version : 0.33.0
* Maven version (`mvn -v`) : 3.3.9
* Docker version : 19.03.12
* If it's a bug, how to reproduce : run docker-maven-plugin:start with a config that waits for a log line

**Comment by rhuss** [comment]
yes, this can be probably improved indeed. happy for any PR tackling this, I probably don't have enough spare room to work on it.

**Comment by naturzukunft** [comment]
i got this error everytime. on:
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal

Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /opt/maven
Java version: 11.0.7, vendor: Oracle Corporation, runtime: xxxxxxxxxxxxxx
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.4.0-51-generic", arch: "amd64", family: "unix"

Is there a workarround?

**Comment by luxmeter** [comment]
Same issue here. My specs:
```
❯ mvn -version
Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T20:33:14+02:00)
Maven home: /usr/local/Cellar/maven/3.5.4/libexec
Java version: 1.8.0_201, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk1.8.0_201.jdk/Contents/Home/jre
Default locale: en_DE, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.6", arch: "x86_64", family: "mac"
```
Docker Desktop version: 2.5.0.1
Docker Engine version: 19.03.13
Maven plugin verison: 0.34.1

Is the issue understood already and is there a workaround as @naturzukunft asked?
@naturzukunft which Docker version do you use?

**Comment by mdsardar** [comment]
Anyone still facing this issue still? Anyone seeing this fixed and any help?
My Specs:
>mvn version 
Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
Maven home: /opt/homebrew/Cellar/maven/3.8.1/libexec
Java version: 1.8.0_292, vendor: Azul Systems, Inc., runtime: /Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/jre
Default locale: en_IN, platform encoding: US-ASCII
OS name: "mac os x", version: "10.16", arch: "x86_64", family: "mac"

**Comment by christopher-nash** [comment]
In case it is helpful to anyone, an engineer on our team was running into this issue on his M1 Mac. It turned out it was just a timeout waiting for the container to launch. **We increased the wait timeout and he was able to move past this issue**. We suspect that it is because the container we were attempting to launch (localstack) was just taking longer to start due to the Rosetta virtualization on the M1 architecture. Thats a band-aid for it, but it at least may get someone over the headache temporarily.
